### PR TITLE
Switch to using official images for containers

### LINKED ISSUE #357: Switch to using official images for containers
**Describe the solution you'd like**
[A clear and concise description of what you want to happen.]
The `node-driver-registrar` and `liveness-probe` container images are currently referenced from mcr for Windows and quay registry for Linux. 

`k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1` and `k8s.gcr.io/sig-storage/livenessprobe:v2.1.0` container images now support multi-arch. We should switch the images in the [manifest_staging](https://github.com/kubernetes-sigs/secrets-store-csi-driver/tree/master/manifest_staging) dir to use the official images for linux and windows.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]
Releases:
https://github.com/kubernetes-csi/livenessprobe/releases/tag/v2.1.0
https://github.com/kubernetes-csi/node-driver-registrar/releases/tag/v2.0.1

**Environment:**

- Secrets Store CSI Driver version: (use the image tag):
- Kubernetes version: (use `kubectl version`):

/good-first-issue
/kind cleanup

### PR DESCRIPTION
**Pull request by Evalle**
**What this PR does / why we need it**:
The `node-driver-registrar` and `liveness-probe` container images are currently referenced from mcr for Windows and quay registry for Linux.

`k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1` and `k8s.gcr.io/sig-storage/livenessprobe:v2.1.0` container images now support multi-arch. This PR switches the images in the manifest_staging dir to use the official images for linux and windows.

**Which issue(s) this PR fixes** 
Fixes #357

**Special notes for your reviewer**:
I can create two new values for general images for mcft and linux in `values.yaml` but I don't know your opinion on that. 
Thank you

### CODE REVIEW

File: manifest_staging/charts/secrets-store-csi-driver/README.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ The following table lists the configurable parameters of the csi-secrets-store-p
...
+| `linux.livenessProbeImage.repository`   | Linux liveness-probe image repository                                                                                             | `k8s.gcr.io/sig-storage/livenessprobe`                                   |
 | `linux.livenessProbeImage.pullPolicy`   | Linux liveness-probe image pull policy                                                                                            | `Always`                                                         |
-| `linux.livenessProbeImage.tag`          | Linux liveness-probe image tag                                                                                                    | `v2.0.0`                                                         |
+| `linux.livenessProbeImage.tag`          | Linux liveness-probe image tag                                                                                                    | `v2.0.1`                                                         |
```
Comment (aramase) [inline manifest_staging/charts/secrets-store-csi-driver/README.md:41]: liveness probe image tag is `v2.1.0`

File: manifest_staging/charts/secrets-store-csi-driver/README.md [hunk 2]
```diff
@@ -51,13 +51,13 @@ The following table lists the configurable parameters of the csi-secrets-store-p
...
-| `windows.livenessProbeImage.repository` | Windows liveness-probe image repository                                                                                           | `mcr.microsoft.com/oss/kubernetes-csi/livenessprobe`             |
+| `windows.livenessProbeImage.repository` | Windows liveness-probe image repository                                                                                           | `k8s.gcr.io/sig-storage/livenessprobe`             |
 | `windows.livenessProbeImage.pullPolicy` | Windows liveness-probe image pull policy                                                                                          | `Always`                                                         |
-| `windows.livenessProbeImage.tag`        | Windows liveness-probe image tag                                                                                                  | `v2.0.1-alpha.1-windows-1809-amd64`                              |
+| `windows.livenessProbeImage.tag`        | Windows liveness-probe image tag                                                                                                  | `v2.0.1`                              |
```
Comment (aramase) [inline manifest_staging/charts/secrets-store-csi-driver/README.md:60]: liveness probe image tag is `v2.1.0`

File: manifest_staging/charts/secrets-store-csi-driver/values.yaml [hunk 3]
```diff
@@ -29,8 +29,8 @@ linux:
         memory: 20Mi
 
   livenessProbeImage:
-    repository: quay.io/k8scsi/livenessprobe
-    tag: v2.0.0
+    repository: k8s.gcr.io/sig-storage/livenessprobe
+    tag: v2.0.1
```
Comment (aramase) [inline manifest_staging/charts/secrets-store-csi-driver/values.yaml:33]: ditto (https://github.com/kubernetes-csi/livenessprobe/releases/tag/v2.1.0)

File: manifest_staging/charts/secrets-store-csi-driver/values.yaml [hunk 4]
```diff
@@ -80,8 +80,8 @@ windows:
         memory: 20Mi
 
   livenessProbeImage:
-    repository: mcr.microsoft.com/oss/kubernetes-csi/livenessprobe
-    tag: v2.0.1-alpha.1-windows-1809-amd64
+    repository: k8s.gcr.io/sig-storage/livenessprobe
+    tag: v2.0.1
```
Comment (aramase) [inline manifest_staging/charts/secrets-store-csi-driver/values.yaml:84]: ditto (https://github.com/kubernetes-csi/livenessprobe/releases/tag/v2.1.0)

**Review by aramase — CHANGES_REQUESTED** [review]

**Comment by Evalle** [comment]
Done, @aramase

**Review by aramase — COMMENTED** [review]
/lgtm

**Pull request merged**